js/src/modals/CreateAccount/NewAccount/newAccount.js

```diff
@@ -43,9 +43,9 @@ export default class CreateAccount extends Component {
     isValidPass: false,
     passwordHint: '',
     password1: '',
-    password1Error: ERRORS.invalidPassword,
+    password1Error: null,
```

Comment:
100% - I have kept all these in for now for circling back. The following to be logged after doing this  -

DRY-up the error checks & validation across the different parts - validation is duplicated in the different sections (Not a critical item, but it is messy atm and can be done much better)